### ISSUE TITLE
Improvement: Session inactive message

### ISSUE THREAD

**Issue by somesh-kumar** [eclipse/vorto#2154]
<img width="790" alt="2019-12-09_17h17_47" src="https://user-images.githubusercontent.com/31053926/70424585-954f8300-1aaa-11ea-83c4-bf30a24a19f1.png">

When my session times out due to inactivity I see a message:-
"Warning - your session has expired. Please save your work and login again."
The message could be misinterpreted. If a user is on the editor page he might just click the save button and loose the changes. 

Expected: If the session has already expired prompt the user to copy the changes elsewhere in a text editor and login again to resume. Show the Login button to take user to the Login page.

**Comment by somesh-kumar** [comment]
Additional Info-

On hitting the save button there is a blank warning message box. Refer to screenshot below. 

Expected: Navigate to login screen in case of session time out

Actual: An empty warning box is shown to user

<img width="915" alt="2019-12-09_18h17_08" src="https://user-images.githubusercontent.com/31053926/70427946-9683ae80-1ab0-11ea-91bb-4008268f5a76.png">

**Comment by mena-bosch** [comment]
Having a look soon.

**Comment by mena-bosch** [comment]
Had a few thoughts about this. 
My idea might seem a bit drastic so @somesh-kumar @kolotu feel free to counter-argue. 

**TL;DR**

1. Make the banner modal so nothing "behind" it can be clicked 
2. Remove the "save your work" part. Unlikely that the user will want to copypaste text from the model's detail and place it somewhere else temporarily, also given the long time of inactivity. 
3. Bottomline, since at that point the session is unrecoverable, the banner should just display the link redirecting to the login page, saying something along the lines of "your session expired, please login again". 

We have a long enough session timeout with warnings in between so losing one's work at this point should be acceptable. 

Will start working in that direction soon if no objections.

**Comment by mena-bosch** [comment]
Elaborating on the above: 

- Grey semi-transparent div covering whole real estate when both warning and session expired banners occur
- The content "behind" it is scrollable but not clickable
- The only way to "un-freeze" the vorto content at this point is to click on the modal's button, whether it's to extend the session or navigate back to the login page depending on the case

Screenshot cutouts below: 

![warning](https://user-images.githubusercontent.com/56636716/72422327-85e6e800-3782-11ea-93b4-0b4e565376cb.png)

![expired](https://user-images.githubusercontent.com/56636716/72422338-8c755f80-3782-11ea-93bc-12e7ecbc7a86.png)

**Comment by mena-bosch** [comment]
PR [here](https://github.com/eclipse/vorto/pull/2228)

**Comment by mena-bosch** [comment]
Merged, @somesh-kumar or @kolotu  cannot close this myself, could you please?

**Comment by kolotu** [comment]
@somesh-kumar  @mena-bosch it's closed
